README.md

```diff
@@ -449,19 +449,17 @@ stringData:
         type: sample-instance-offering, // The service offering name
     }
 ```
-###### Custom Templates
+##### Custom Templates
```

Comment:
"Custom Secret Format"

u dont customize the template, the template is the *mean* to customize how the secret would look like.